### DISCUSSION TITLE
Glitching Information Button On Two-Factor Authentication Preferences

### DISCUSSION THREAD

**Discussion by dlahmanchuk** [standardnotes/forum#3009] [General]
**Bug Description:**
Whenever I click on the information button on Two-Factor Authentication, it glitches very quickly, and I can't read it.

**To Reproduce**
1. Go to Preferences
2. Click on Security
3. Scroll down to Two-Factor Authentication
4. Click on The Information Button

**Comment by effieeee** [comment]
Hi @Daniel-Lahmanchuk, I wanted to ask if you still encounter this issue until now? Can you also let me know your current app platform (desktop, mobile, or web), and which version of Standard Notes you're using at the moment?

**Comment by dlahmanchuk** [comment]
Hello @effieeee, I checked the issue and it's still happening today. I'm currently using the web to use Standard Notes and I'm currently on the version 3.151.2
